### PR TITLE
feat

### PR DESCRIPTION
**Pull request by ckahub**
#Cart
added cart page to ecommerc
chrome-capture-2022-2-26 (3)
e

### CODE REVIEW

**Review by dusariudaykumar — APPROVED** [review]
remove empty files.
looks good

File: style/login.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    left: 0;
+    top: 1rem;
+    display: none;;
+}
+.nav2 ul ul li{
+    background-color: tan;
+    border-bottom: 0.1rem solid white;
+    color: var(--primary-color);
+}
+.nav2 ul ul li a{
```
Comment (ravikumar0403) [inline style/login.css:61]: try not to use html tags for editing and avoid using multiple tags because it increases specificity

**Comment by manishsom20** [comment]
No changes required
I like your code structure, naming convention and usage of utility classes

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    <title>Ethinic Wear</title>
+</head>
+<body>
+    <header>
+        <!-----nav1-start-->
+        <nav class="navigation">
+            <ul class="left_Brand">
+                <a href="./homepage.html">Ethinic Wear</a>
+            </ul>
+            <ul class="right_Brand">
```
Comment (ravikumar0403) [inline pages/login.html:20]: try to maintain consistent naming convention

**Comment by ckahub** [comment]
thanx uday and manish for review

**Review by ravikumar0403 — APPROVED** [review]
Looks good. I've left a couple of comments

**Pull request merged**